src/clj/fluree/db/json_ld/credential.cljc

```diff
@@ -115,8 +115,22 @@
         auth-did                 (did/auth-id->did id)]
     {:subject (json/parse payload false) :did auth-did}))
 
+(defn is-JWS?
+  [signed-transaction]
+  (string? signed-transaction))
```

Comment:
A problem Jace ran across is SPARQL queries are just strings, so we can't tell the difference between a SPARQL query and a JWS with just the `string?` predicate. 

This regex should work: `#"(^[A-Za-z0-9-_]*\.[A-Za-z0-9-_]*\.[A-Za-z0-9-_]*$)"`.

Reply:
Ahh thank you! I actually ran into this issue with fluree/server and didn't know why and haven't had time to dig in. That saved me a bunch of time I suspect.

Reply:
I've really been thinking SPARQL should have its own API... but I am also cognizant that the API surface is growing.